kamil.sharipov/task-2-2/internal/intheap/intheap.go

```diff
@@ -0,0 +1,50 @@
+package intheap
+
+import (
+	"container/heap"
+)
+
+type IntHeap []int
+
+func (h IntHeap) Len() int {
+	return len(h)
+}
+
+func (h IntHeap) Less(i, j int) bool {
+	return h[i] > h[j]
+}
+
+func (h IntHeap) Swap(i, j int) {
+	h[i], h[j] = h[j], h[i]
+}
+
+//nolint:recvcheck
+func (h *IntHeap) Push(x interface{}) {
+	rating, ok := x.(int)
+	if !ok {
+		panic("IntHeap.Push: expected int")
```

Comment:
это контекст мы явно получим из стектрейса... можно проще
```suggestion
		panic("push into heap: expected int value")
```